css/info.css

```diff
@@ -0,0 +1,69 @@
+.CodeMirror-info {
```

Comment:
Just wondering if we should have CSS rules specific to `codemirror-graphql` in that repo, wdyt?